app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(10, 3)
```

Comment:
The checklist advises against creating unnecessary variables for simple tests. You can directly assert the result of `split_integer(10, 3)` without assigning it to a variable.